doc/source/getting_started/docker.rst

```diff
@@ -124,3 +129,123 @@ with the `-np` switch.  For example:
 For additional command line arguments please see the ansys
 documentation at `ANSYS help <https://ansyshelp.ansys.com>`_.  Also,
 be sure to have the appropriate license for additional HPC features.
+
+Using ``--restart`` policy with MAPDL products
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, MAPDL creates a ``LOCK`` file at the working directory when starts
+and deletes it if it exits normally. This file is used to avoid overwriting the
+files such as database files (``db``) or result files (``rst``) when starting
+MAPDL after an abnormal termination.
+
+Because of this behaviour, when using the docker flag ``--restart`` in ``docker run``,
+you might enter in an infinite loop if you specify the docker image to reboot after an
+abnormal termination (i.e. ``--restart always``). 
+Because of the presence of the ``LOCK``, MAPDL exits attempting to not overwrite
+the files from the previous crash, while the docker process keeps attempting to
+restart the MAPDL container (and the MAPDL process with it).
```

Comment:
```suggestion
Because of this behavior, when using the Docker ``--restart`` flag in the ``docker run``
command, you might enter into an infinite loop if you specify the Docker image to
reboot after an abnormal termination. For example, ``--restart always``. 
Because of the presence of the ``LOCK`` file, MAPDL exits, attempting to not overwrite
the files from the previous crash, while the Docker process keeps attempting to
restart the MAPDL container (and the MAPDL process with it).
```